cmake/rt-kernel.cmake

```diff
@@ -45,6 +45,21 @@ target_sources(pn_dev
   sample_app/sampleapp_common.c
   src/ports/rt-kernel/sampleapp_main.c
   )
+# Generate version numbers
```

Comment:
rt-kernel has a built-in version string but unfortunately there is no api for it. Try e.g.:

```
extern char * os_kernel_revision;
printf ("build: %s\n", os_kernel_revision);

```

Reply:
Done